### PR TITLE
Allow to create multiple XR projection layers

### PR DESCRIPTION
**Pull request by hmiguellima**
**Description**

Since the recent XRLayers spec was made available, it's been technically possible to create immersive media player web apps with complex user interfaces, like for example for a scenario where the user is inside a VR cinema room with several 3D assets (not possible to convincingly recreate using a simple cube map or equirect  layer) rendered on a background layer, followed by a video layer, and on top, an user interface layer with controllers/hands and simple UI for player control.

This change exposes the newly created XR layer to the app, through a new method  `getWebGLLayer` on the renderer's XR manager, that can be called after the manager's `setSession` method resolves, providing a way to create complex multi layer media apps.

Here's an example app I made to demo this feature:
https://github.com/hmiguellima/cinema-room

I've ran the above demo on a Quest 2 with acceptable frame rates using the new fixedFoveation option of the XRLayer.

### CODE REVIEW

File: src/renderers/webxr/WebXRManager.js [hunk 1]
```diff
@@ -270,6 +270,8 @@ class WebXRManager extends EventDispatcher {
 
 				scope.dispatchEvent( { type: 'sessionstart' } );
 
+				return glProjLayer || glBaseLayer;
```
Comment (mrdoob) [inline src/renderers/webxr/WebXRManager.js:273]: How about adding a `getWebGLLayer()` method to `WebXRManager` instead?

File: examples/jsm/webxr/VRButton.js [hunk 2]
```diff
@@ -18,7 +18,11 @@ class VRButton {
 
 				session.addEventListener( 'end', onSessionEnded );
 
-				await renderer.xr.setSession( session );
+				if (typeof rendererOrCallback === 'function') {
+					await rendererOrCallback( session );
+				} else {
+					await renderer.xr.setSession( session );
+				}
```
Comment (mrdoob) [inline examples/jsm/webxr/VRButton.js:25]: Can you explain how does this change relate to this PR?

File: src/renderers/webxr/WebXRManager.js (hunk 1 above)
Comment (hmiguellima) [reply src/renderers/webxr/WebXRManager.js:273]: Makes sense and would be more in line in the rest of the API. I'll change it.

File: examples/jsm/webxr/VRButton.js (hunk 2 above)
Comment (mrdoob) [inline examples/jsm/webxr/VRButton.js:25]: Is this change still needed?
